From e7134d551300f62183c2fda49af9d2dd2a4dc089 Mon Sep 17 00:00:00 2001
From: maxmaster <yousseflkashef000@gmail.com>
Date: Mon, 1 Jun 2026 10:32:00 +0000
Subject: [PATCH] makefile yocto fix

---
 makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/makefile b/makefile
index 44ce757..713aeee 100644
--- a/makefile
+++ b/makefile
@@ -1,5 +1,5 @@
 all: dash.c
-	cc  -Wall -o dash  dash.c
+	${CC} ${CFLAGS} ${LDFLAGS} -o dash  dash.c
 
 clean:
 	dash
-- 
2.43.0

